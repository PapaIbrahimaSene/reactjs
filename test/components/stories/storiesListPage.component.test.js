import React from 'react';
import { shallow } from 'enzyme';
import { StoriesListPage } from '../../../src/components/stories/storiesListPage.component';

test('should create StoriesList page', () => {
    const wrapper = shallow(<StoriesListPage />);

    expect(wrapper).toMatchSnapshot();
});