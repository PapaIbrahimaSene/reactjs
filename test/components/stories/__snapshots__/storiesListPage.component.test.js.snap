// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should create StoriesList page 1`] = `
<div>
  <MainMenu />
  <Connect(StoriesList) />
</div>
`;
